ssh-key-algo

```diff
@@ -31,7 +31,7 @@ then
   exit 0
 fi
 
-echo "using: $(which ssh)"
+echo "using: $(command -v ssh)"
```

Comment:
You may want to use `command -pv` to bypass alias. Then later on, use `command -p ssh -vvv ...` instead of just `ssh -vvv`

Reply:
Perhaps ... but I would imagine the typical use case is to test the user's typical setup, so if they've aliased `ssh` intentionally or have a non-default `$PATH`, I would think we'd want to check using those configurations.